src/main/webapp/WEB-INF/tags/instructor/courseStudentDetailsEdit/studentInformationTable.tag

```diff
@@ -9,6 +9,8 @@
     <div class="panel-body fill-plain">
         <form action="<%=Const.ActionURIs.INSTRUCTOR_COURSE_STUDENT_DETAILS_EDIT_SAVE%>" method="post" id="instructor-student-edit-form" class="form form-horizontal">
             <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>" value="${studentInfoTable.course}">
+            <input type="hidden" name="<%=Const.ParamsNames.EMAIL_SEND_CHECK%>" value="false">
+            <input type="hidden" id="openorpublishedemailsent" name="openorpublishedemailsent" value="${openOrPublishedEmailSentForTheCourse}">
```

Comment:
Also, `value="{openOrPublishedEmailSentForTheCourse}"` does not seem right. You would see in the HTML test file that this converts to `value=""` which should not be the case.